<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录制屏幕</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="startBtn">
         <property name="text">
          <string>开始录制</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectRectBtn">
         <property name="text">
          <string>选择录制区域</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopBtn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止录制</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="useLogoCheckBox">
         <property name="text">
          <string>使用Logo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="HOffsetCheckBox">
           <property name="text">
            <string>logo水平偏移</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="HOffsetSpinBox">
           <property name="maximum">
            <number>1080</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="VOffsetCheckBox">
           <property name="text">
            <string>logo垂直偏移</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="VOffsetSpinBox">
           <property name="maximum">
            <number>1920</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="hideCheckBox">
         <property name="text">
          <string>自动隐藏</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="infoCheckBox">
         <property name="text">
          <string>完成提示</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="passCheckBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>压缩</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>保存路径</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>./Save/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="randBtn">
       <property name="text">
        <string>随便</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pathBtn">
       <property name="text">
        <string>更改</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
